public/themes/custom/hdbt_subtheme/hdbt_subtheme.theme

```diff
@@ -128,11 +128,14 @@ function hdbt_subtheme_form_views_exposed_form_alter(&$form, $form_state) {
     return;
   }
 
-  if ($paragraph->hasField('field_hs_search_meta_button')) {
+  if ($paragraph->hasField('field_hs_search_meta_button') && $paragraph->get('field_hs_search_meta_button')->value) {
```

Comment:
Should probably use `!$paragraph->get('field_hs_search_meta_button')->isEmpty()` here.